CONTRIBUTING.md

```diff
@@ -39,22 +37,22 @@ Opening an issue beforehand allows the administrators and the community to discu
 6. PR of reformatting (changing of ws/TAB, line endings or coding style) of source code won't be accepted. Use issue trackers for your request instead.
 7. Typo fixing and code refactoring won't be accepted - please create issues with title started with `TYPO` to request the changing.
 8. Address the review change requests by pushing new commits to the same PR. Avoid amending a commit and then force pushing it. All the PR commits are squashed before merging to the main branch.
-9. Finally, please test your pull requests, at least once.
+9. When creating new PR, try to base it on latest master.
+10. Don't merge upstream/master (using git or via github sync), if your PR is based on older master. If you need to base it on latest master (e.g. to check and fix merge conflict), use commands `git fetch upstream` to get latest "master" and then `git rebase upstream/master` to rebased it on latest master.
```

Comment:
What about `git pull --rebase upstream master`?

Reply:
@Mateos81 Nope.
I've still got the conflict with such way, IIRC.
So let's following the way which is stable.

Reply:
@Mateos81 
as @donho said there might be sometimes conflict with it, 
IIRC it is combination of `git fetch upstream` and `git rebase --onto <some 2-3 branches, I don't remember order>`

but thank to this I've noticed typo in text